src/main/java/Duke.java

```diff
@@ -1,5 +1,18 @@
 import java.io.*;
```

Comment:
Maybe you could change this to explicitly importing only the necessary packages?